tests/functional/pv/space_reclaim/test_rbd_reclaimspace_cronjob.py

```diff
@@ -98,10 +98,11 @@ def test_rbd_space_reclaim_cronjob(self, pause_and_resume_cluster_load):
             )
             pod_obj.get_fio_results()
 
-        # Verify used size after IO
-        exp_used_size_after_io = used_size_before_io + (16 * self.pool_replica)
-        used_size_after_io = fetch_used_size(cbp_name, exp_used_size_after_io)
-        log.info(f"Used size after IO is {used_size_after_io}")
+            # Verify used size after IO
+            exp_used_size_after_io = used_size_before_io + (4 * self.pool_replica)
```

Comment:
How does the new calculation makes a difference in the final value of ```used_size_before_io``` ? I am trying to understand how the new calculation solves the issue ?

Reply:
> How does the new calculation makes a difference in the final value of `used_size_before_io` ? I am trying to understand how the new calculation solves the issue ?

When we calculate the used size of the pool, the negligible difference between the expected size and actual size is 1.5Gb, but in my case sometimes, very rarely, it exceeds 1.5Gb to 2Gb, because of that script will fail. In this test we calculate the pool size after creating 4 files cumulatively.  So I assumed, calculating the pool size after each file creation would resolve the issue.  If not this way, we need to make changes in this function _fetch_used_size_ . Please suggest if you think otherwise.